packages/augur-ui/src/modules/auth/components/connect-account/connect-account.tsx

```diff
@@ -5,26 +5,41 @@ import Blockies from 'react-blockies';
 import ConnectDropdown from 'modules/auth/containers/connect-dropdown';
 import ChevronFlip from 'modules/common/chevron-flip';
 import formatAddress from 'modules/auth/helpers/format-address';
+import { LoginAccount } from 'modules/types';
 
 import Styles from 'modules/auth/components/connect-account/connect-account.styles.less';
 import ToggleHeightStyles from 'utils/toggle-height.styles.less';
-import { LoginAccount } from 'modules/types';
 
 interface ConnectAccountProps {
   isLogged: boolean;
   isConnectionTrayOpen: boolean;
   updateConnectionTray: Function;
+  updateMobileMenuState: Function;
+  isMobile: boolean;
```

Comment:
Need to use media query, dont want to keep using isMobile

Reply:
refactored away 👍